aten/src/ATen/native/Integration.cpp

```diff
@@ -0,0 +1,72 @@
+#include <ATen/ATen.h>
+#include <ATen/NativeFunctions.h>
+#include <ATen/WrapDimUtils.h>
+#include <ATen/core/DimVector.h>
+#include <c10/util/Exception.h>
+
+namespace at {
+namespace native {
+namespace {
```

Comment:
(c++ question) what is the advantage of putting static functions into an anonymous namespace? Isn't it sufficient to make them non-static (the namespace is anonymous so they can't be accessed from outside this file?))

Reply:
I think you are right, idk why I made it static